addons/sourcemod/scripting/gokz-jumpstats/jump_tracking.sp

```diff
@@ -283,21 +284,22 @@ enum struct JumpTracker
 		}
 		else if (ladderJump)
 		{
-			if (this.tickCount - this.lastJumpTick <= JS_MAX_BHOP_GROUND_TICKS)
+			// Check for ladder gliding.
+			float curtime = GetGameTime();
+			float ignoreLadderJumpTime = GetEntPropFloat(this.jumper, Prop_Data, "m_ignoreLadderJumpTime");
+			// Check if the ladder glide period is still active and if the player held jump in that period.
+			if (ignoreLadderJumpTime > curtime &&
+				ignoreLadderJumpTime - 0.2 < lastJumpButtonTime[this.jumper] && lastJumpButtonTime[this.jumper] < ignoreLadderJumpTime)
```

Comment:
Is the 0.2 a specific value or was it determined by trial-and-error? Should probably have a `#define` for it regardless I think.

Reply:
0.2 is a specific value in the game code yes